Description: ui is not present / ready yet. Remove the build tag so that it will
not try to embed the ui.
Upstream: no
diff --git a/Makefile b/Makefile
index 7111f9e..ab72a2e 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ default: dev
 
 # bin generates the releasable binaries for OpenBao
 bin: prep
-	@CGO_ENABLED=$(CGO_ENABLED) BUILD_TAGS='$(BUILD_TAGS) ui' sh -c "'$(CURDIR)/scripts/build.sh'"
+	@CGO_ENABLED=$(CGO_ENABLED) BUILD_TAGS='$(BUILD_TAGS)' sh -c "'$(CURDIR)/scripts/build.sh'"
 
 # dev creates binaries for testing OpenBao locally. These are put
 # into ./bin/ as well as $GOPATH/bin
